### PR TITLE
Store responses for web and iOS

### PR DESCRIPTION
**Pull request by cking0987**
@TheOmnimax can you review this and test it out before merging?

### CODE REVIEW

**Comment by TheOmnimax** [comment]
I tested on web forms and iOS. On web forms, you can just click the button, and a response is stored.

However, on iOS, no response is stored at all. Although, I don't know if that is because I tried calling with Facetime since my iPad is not a phone.

File: README.md [hunk 1]
```diff
@@ -10,17 +10,25 @@ This field plug-in can dial a phone number for you in SurveyCTO Collect on Andro
...
-    > [TIMESTAMP] Failure calling the following phone number: [phone_number].
+* Supports the `text` field type, but doesn't provide an actual text input. 
+
+## How the response is stored
+
+> [`TIMESTAMP`] The following phone number was called:  `phone_number`.
+
+* This response will use the `phone_number` value from the parameter, and `TIMESTAMP` will be the current date and time.
+* If you're using iOS or web forms, this response will be appended to the response (on a new line) whenever the main button is clicked.
+* If you're using Android, this response will be appended to the response (on a new line) if the dialer app was successfully launched.
```
Comment (TheOmnimax) [inline README.md:21]: Is "appended" the right word? That makes it seem like it is added on. Maybe it should be, "the response will be stored".

**Comment by TheOmnimax** [comment]
I believe the code itself is good, but I'll take another look tomorrow.

File: README.md (hunk 1 above)
Comment (cking0987) [reply README.md:21]: Yes, appended is the right word. If you look at the code, it adds that line to the end of the existing stored response. This is to be able to record when multiple phone calls are made from the same field (maybe there were two errors, then the third time worked).

**Pull request merged**